docs/resources/artifactory_access_token.md

```diff
@@ -0,0 +1,173 @@
+# Artifactory Access Token Resource
+
+Provides an Artifactory Access Token resource. This can be used to create and manage Artifactory Access Tokens.
+
+~> **Note:** Access Tokens will be stored in the raw state as plain-text. [Read more about sensitive data in
+state](https://www.terraform.io/docs/state/sensitive-data.html).
+
+
+## Example Usages
+### Create a new Artifactory Access Token for an existing user
+```hcl
+resource "artifactory_access_token" "exising_user" {
```

Comment:
You have a typo in your resource declaration - Once I changed this (in my test script), it seemed to work.